config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
   mount_devise_token_auth_for 'User', at: 'login'
-  resources :books, only: [:index, :show]
+  resources :books, only: [:index, :show] 
+  resources :users do
```

Comment:
This creates all the possible routes for users (check `rails routes`). You probably need an only: []